charts/pg-operator/values.yaml

```diff
@@ -0,0 +1,140 @@
+---
+# ======================
+# Installer Controls
+# ======================
+fullnameOverride: ""
+
+# rbac: settings for deployer RBAC creation
+rbac:
+    # rbac.create: if false RBAC resources should be in place
+    create: true
+    # rbac.useClusterAdmin: creates a ClusterRoleBinding giving cluster-admin to serviceAccount.name
+    useClusterAdmin: false
+
+# serviceAccount: settings for Service Account used by the deployer
+serviceAccount:
+    # serviceAccount.create: Whether to create a Service Account or not
+    create: true
+    # serviceAccount.name: The name of the Service Account to create or use
+    name: ""
+
+archive_mode: "true"
+archive_timeout: "60"
+backrest_aws_s3_bucket: ""
+backrest_aws_s3_endpoint: ""
+backrest_aws_s3_key: ""
+backrest_aws_s3_region: ""
+backrest_aws_s3_secret: ""
+backrest_aws_s3_uri_style: ""
+backrest_aws_s3_verify_tls: "true"
+backrest_gcs_bucket: ""
+backrest_gcs_endpoint: ""
+backrest_gcs_key_type: ""
+backrest_port: "2022"
+badger: "false"
+ccp_image_prefix: "perconalab/percona-postgresql-operator"
+ccp_image_pull_secret: ""
+ccp_image_pull_secret_manifest: ""
+ccp_image_tag: "main-postgres-ha"
```

Comment:
I think we need to add chart for 0.1.0 for now

Reply:
done